CMakeLists.txt

```diff
@@ -188,6 +188,9 @@ else()
         find_package( hip REQUIRED CONFIG PATHS ${HIP_DIR} ${ROCM_PATH} /opt/rocm )
     endif( )
 
+    # Add rocisa
+    add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/tensilelite/rocisa)
```

Comment:
Style nit: typically leave off ${CMAKE_CURRENT_SOURCE_DIR} for add_subdirectory since it is the default.